crates/hir-expand/src/builtin_fn_macro.rs

```diff
@@ -90,11 +94,6 @@ register_builtin! {
     (module_path, ModulePath) => module_path_expand,
     (assert, Assert) => assert_expand,
     (stringify, Stringify) => stringify_expand,
-    (format_args, FormatArgs) => format_args_expand,
-    (const_format_args, ConstFormatArgs) => format_args_expand,
-    // format_args_nl only differs in that it adds a newline in the end,
-    // so we use the same stub expansion for now
-    (format_args_nl, FormatArgsNl) => format_args_expand,
```

Comment:
Just a sanity check but these *used* to be lazy right?

Reply:
Ye, I checked that its eager in rustc as well. But I'm surprised we had this wrong since the beginning then...

Reply:
Just checked history, it was in fact always eager, so we had it wrong this entire time